src/utils/image/index.ts

```diff
@@ -12,21 +12,49 @@ const compressAndPrepareFormData = async (imagePath: string) => {
 };
 
 const uploadImage = async (imagePath: string) => {
-  const formData = await compressAndPrepareFormData(imagePath);
+  try {
+    const formData = await compressAndPrepareFormData(imagePath);
+
+    const imageUrl = await uploadPhoto(formData);
+    return imageUrl;
+  } catch (error) {
+    console.log(error);
+    return imagePath;
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `uploadImage` function could be improved. Currently, it just logs the error and returns the original image path. This might not be the expected behavior when an upload fails. Consider throwing the error to allow the calling function to handle it.

```diff
-    console.log(error);
-    return imagePath;
+    throw error;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->